Strategic-Initiatives.md

```diff
@@ -53,8 +52,10 @@ and have the support needed.
 [node#18106]: https://github.com/nodejs/node/issues/18106
 [fishrock123]: https://github.com/fishrock123
 [addaleax]: https://github.com/addaleax
+[cclauss]: https://github.com/cclauss
```

Comment:
Isn't needed now that initiative is removed

Reply:
good catch, thanks.